fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -113,10 +114,11 @@ public void cancelLoadJob(CancelLoadStmt stmt) throws DdlException {
             if (!dbIdToLabelToLoadJobs.containsKey(db.getId())) {
                 throw new DdlException("Load job does not exist");
             }
-            if (!dbIdToLabelToLoadJobs.get(db.getId()).containsKey(stmt.getLabel())) {
+            Map<String, List<LoadJob>> labelToLoadJobs = dbIdToLabelToLoadJobs.get(db.getId());
+            if (!labelToLoadJobs.containsKey(stmt.getLabel())) {
                 throw new DdlException("Load job does not exist");
             }
-            List<LoadJob> loadJobList = dbIdToLabelToLoadJobs.get(db.getId()).get(stmt.getLabel());
+            List<LoadJob> loadJobList = labelToLoadJobs.get(stmt.getLabel());
```

Comment:
why don't you get and check if the result is null

Reply:
Because I think `containsKey` is more graceful then checking the result is null is more

Reply:
I don't agree with you, you get it twice. And in some cases, If another thread modify this map, your check can't make sure you can get a not null value

Reply:
Why you two hold opposite opinions /(ㄒoㄒ)/~~ @morningman

Reply:
I do not get it twice. The containsKey does not get the value, it just hash the key. Also, I add the read lock whether get and check null or containsKey

Reply:
Hash the key is the first operation, `containsKey` will also check if two keys are equal, because hash of two keys may equal. And these operation's cost is less than a `get` operation.

Yes, in this case you add a readLock. But what I want to say is that it is good habit to check get's value

Reply:
Yes, you are right. If I only check the key, the containsKey is better. If I need to check and get in concurrent situation, the get and check null is better.